massa-pos-worker/src/controller.rs

```diff
@@ -157,16 +155,8 @@ impl SelectorController for SelectorControllerImpl {
     /// Get every [Selection]
     ///
     /// Only used in tests for post-bootstrap selection matching.
-    #[cfg(feature = "testing")]
     fn get_entire_selection(&self) -> VecDeque<(u64, HashMap<Slot, Selection>)> {
-        let (_, lock) = &*self.cache;
-        let cache_guard = lock.read();
-        let cache = cache_guard.as_ref().map_err(|err| err.clone()).unwrap();
-        cache
-            .0
-            .iter()
-            .map(|cycle_draws| (cycle_draws.cycle, cycle_draws.draws.clone()))
-            .collect()
+        panic!("Method should only be used by a mock of the selector controller.");
```

Comment:
We are using it in the tests of the bootstrap and we don't wanna use a mock right there

Reply:
Thanks for the info, I've re-added the method body, leaving the feature flag out. By the way it seems bootstrap doesn't use that feature, and the compilation problem only occurred when it is used. We can fix it with the other issue...